src/pudl_rmi/connect_deprish_to_eia.py

```diff
@@ -258,7 +258,7 @@ def add_overrides(deprish_match, file_path_deprish, sheet_name_output):
             .any(axis="columns")
         ][
             RESTRICT_MATCH_COLS
-            + ["plant_part_name", "data_source"]
+            + ["plant_part_name", "data_source", "line_id"]
```

Comment:
Does IDX_COLS already exist? Or would you have to create it for this? Seems like if RESTRICT_MATCH_COLS exists it makes sense to create an IDX_COLS variable because these same columns are referenced down below.